allennlp/modules/token_vectorizer.py

```diff
@@ -0,0 +1,30 @@
+import torch
+
+from allennlp.common import Params
+from allennlp.data import Vocabulary
+
+class TokenVectorizer(torch.nn.Module):
+    """
+    A ``TokenVectorizer`` is a ``Module`` that takes as input a tensor with integer ids that have
+    been output from a :class:`~allennutlp.data.TokenIndexer` and outputs a vector per token in the
```

Comment:
Thanks.  I was having this really weird problem where my internet connection was dropping packets, and my ssh connection I _think_ was resending characters from the line below where my cursor was, so I would get these really weird input strings.  It was pretty frustrating =).  I thought I caught all of the slip ups, but it looks like I missed one.